examples/fixed-known-problems/rewriterIssue4.pvl

```diff
@@ -1,7 +1,7 @@
 // -*- tab-width:2 ; indent-tabs-mode:nil -*-
 
 //:: cases RW4
-//:: suite skip-travis
+//:: suite problem-fixed
```

Comment:
Is the problem-fixed suite run on the CI?

Reply:
Looks like it since only suites are excluded, but just want to doublecheck if you agree

Reply:
Well actually only RW 1,2,3, and 7 are run. 4, 5 and 6 are missing. I am not sure why.

Reply:
I found them and they are running in the CI.